tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/CircuitBreakerRetryTest.java

```diff
@@ -105,7 +118,7 @@ public void testCircuitOpenWithMoreRetries() {
         catch (Exception ex) {
             // Not Expected
             invokeCounter = clientForCBWithRetry.getCounterForInvokingServiceA();
-            Assert.fail("serviceA should retry or throw a CircuitBreakerOpenException in testCircuitOpenWithMoreRetries on iteration "
+            Assert.fail("serviceA should retry or throw a CircuitthenBreakerOpenException in testCircuitOpenWithMoreRetries on iteration "
```

Comment:
Typo, should probably be reverted to previous version.

Reply:
Not sure how that slipped in